mutest.gemspec

```diff
@@ -39,8 +39,4 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency('anima',         '~> 0.3.0')
   gem.add_runtime_dependency('concord',       '~> 0.1.5')
   gem.add_runtime_dependency('regexp_parser', '~> 0.4.1')
-
-  gem.add_development_dependency('rubocop-rspec', '~> 1.10.0')
```

Comment:
It seems to me like it'd be better to have all our dependencies here and just do a `gemspec` in the Gemfile. Something seems weird with the Gemfile though in this project since it won't show a text diff.

Also: `rubocop`, etc. seem to be outside of a group. Seems like they should be in `development` and/or `test` if they're going to be there?